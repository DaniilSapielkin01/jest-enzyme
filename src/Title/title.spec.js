import React from "react";
import Title from "./title";

const setUp = (props) => shallow(<Title {...props} />);

describe("Title component", () => {
  it("shoud render Title with props", () => {
    expect(setUp({ title: "My title" })).toMatchSnapshot();
  });
  it("shoud render Title without props", () => {
    expect(setUp()).toMatchSnapshot();
  });
});
