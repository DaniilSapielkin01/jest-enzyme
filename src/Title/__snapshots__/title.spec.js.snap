// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component shoud render Title with props 1`] = `
<h1
  className="title"
>
  My title
</h1>
`;

exports[`Title component shoud render Title without props 1`] = `
<h1
  className="title"
>
  Simple title
</h1>
`;
